e2e/decomission/decomission_test.go

```diff
@@ -0,0 +1,97 @@
+/*
+Copyright 2021 The Cockroach Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    https://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package decommission
+
+import (
+	"flag"
+	"os"
+	"testing"
+	"time"
+
+	"github.com/cockroachdb/cockroach-operator/pkg/actor"
+	"github.com/cockroachdb/cockroach-operator/pkg/controller"
+	"github.com/cockroachdb/cockroach-operator/pkg/testutil"
+	testenv "github.com/cockroachdb/cockroach-operator/pkg/testutil/env"
+	"github.com/go-logr/zapr"
+	"github.com/stretchr/testify/require"
+	"go.uber.org/zap/zaptest"
+)
+
+// TODO parallel seems to be buggy.  Not certain why, but we need to figure out if running with the operator
+// deployed in the cluster helps
+// We may have a threadsafe problem where one test starts messing with another test
+var parallel = *flag.Bool("parallel", false, "run tests in parallel")
+
+// run the pvc test
+var pvc = flag.Bool("pvc", false, "run pvc test")
+
+// TODO should we make this an atomic that is created by evn pkg?
+var env *testenv.ActiveEnv
+
+// TestMain wraps the unit tests. Set TEST_DO_NOT_USE_KIND evnvironment variable to any value
+// if you do not want this test to start a k8s cluster using kind.
+func TestMain(m *testing.M) {
+	e := testenv.CreateActiveEnvForTest([]string{"..", ".."})
+	env = e.Start()
+	code := testenv.RunCode(m, e)
+	os.Exit(code)
+}
+
+func TestDecommissionFunctionality(t *testing.T) {
+
+	// Testing removing and decommisioning a node.  We start at 4 node and then
+	// remove the 4th node
+
+	if testing.Short() {
```

Comment:
if we're breaking this up, shouldn't this be caught before we ever get this far? I don't think the decommission test does anything in short mode

Reply:
You can still run bazel test //e2e/... and then choose which tests to skip.  I use the skip to isolate a test because I can never remember the full option to just run one test name in go. So for developer friendliness can we leave it in there.